__tests__/app.test.js

```diff
@@ -190,3 +193,56 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH: /api/articles/:article_id", () => {
+  test("POST: 200 should update an article by article_id.", () => {
+    const vote = { inc_votes: 50 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(vote)
+      .expect(200)
+      .then((res) => {
+        const article = res.body;
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("article_id");
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
+
+        expect(article.votes).toBe(150);
+      });
+  });
+  test("POST: 404 responds with a 404 status code if id is not found", () => {
+    const vote = { inc_votes: 50 };
+    return request(app)
+      .patch("/api/articles/123456789")
+      .send(vote)
+      .expect(404)
+      .then(({ res }) => {
+        expect(res.statusMessage).toBe("Not Found");
+      });
+  });
+  test("POST: 400 responds with a 400 status code if invalid id is given", () => {
+    const vote = { inc_votes: 50 };
+    return request(app)
+      .patch("/api/articles/six")
+      .send(vote)
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("invalid request");
+      });
+  });
+  test("POST: 400 responds with a 400 status code if inc_votes is not a number", () => {
+    const vote = { inc_votes: "fifty" };
+    return request(app)
+      .patch("/api/articles/123456789")
+      .send(vote)
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("invalid request");
+      });
+  });
+});
```

Comment:
The above test is great but a further error test would be if the request body is missing the required property altogether